support-models/src/main/scala/com/gu/support/workers/AppliedPromotion.scala

```diff
@@ -3,7 +3,7 @@ package com.gu.support.workers
 import com.gu.support.promotions.PromoCode
 import io.circe.Codec
 
-case class AppliedPromotion(promoCode: PromoCode, countryGroupId: String)
+case class AppliedPromotion(promoCode: PromoCode, countryGroupId: String, supportRegionId: Option[String])
```

Comment:
Interested in the reasoning behind adding this as an `Option` here vs not adding it at all (I'm assuming the `Option` is removed in a future PR, I guess it could just be added in that future PR?) I haven't got a strong reason to prefer one or the other, but asking in case there are elements to this that I'm missing.

Reply:
That's a really good point actually, I hadn't thought that we don't need to use the value yet, so we don't need to add it to this class